lms/templates/student_account/account_settings.underscore

```diff
@@ -0,0 +1,25 @@
+<div class="account-settings">
+    <div class="account-settings-header">
+        <div class="account-settings-header-title">
+            <%- gettext("Account Settings") %>
+        </div>
+        <small class="account-settings-header-subtitle"><%- gettext("A collection of all your account settings, including basic personalization controls, demographic information, and linked social accounts.") %></small>
```

Comment:
Text:

These settings include basic information about your account. You can also specify additional information and see your linked social accounts on this page.

Comment: I'd like to avoid the word "demographic" for a couple of reasons: It's a bit formal, and, to me at least, it has a "we're collecting data on you" connotation.